components/layout/LocaleDropdown.js

```diff
@@ -18,17 +21,24 @@ const LocaleDropdown = () => {
         appearance: 'none',
         WebkitAppearance: 'none',
         borderRadius: '0',
-        border:'0',
-        outline:'none',
-        borderLeft: '1px solid #aeb1be',
-        borderRight: '1px solid #aeb1be'
+        border: '0',
+        outline: 'none',
+        borderLeft: '1px solid #aeb1be'
     };
 
+    const [loading, languages] = useDotCMSApi(getLanguages);
```

Comment:
What's the idea of using this `hook` if you're not using `loading`?

Reply:
feels like hook  is not needed.

Reply:
In React there are different ways of doing request in the client side, my idea with this hook is to have a unique way of doing it in this project, even tho I'm not using the `loading` in this particular case it could be helpful also we might want to add an `error` to so we can fallback if the request fails.